code/modules/hydroponics/seeds.dm

```diff
@@ -386,11 +386,11 @@
 	growthstages = 4
 
 /obj/item/seeds/icepepperseed
-	name = "pack of ice-pepper seeds"
-	desc = "These seeds grow into ice-pepper plants."
+	name = "pack of chilly pepper seeds"
+	desc = "These seeds grow into chilly pepper plants."
 	icon_state = "seed-icepepper"
 	species = "chiliice"
-	plantname = "Ice-Pepper Plants"
+	plantname = "Chilly Pepper Plants"

```

Comment:
Put it back goof.

It's not that funny.

It's just mildly annoying for botanists with lots of seeds.